src/Sulu/Bundle/PageBundle/Resources/views/Export/Snippet/1.2.xliff.twig

```diff
@@ -8,36 +8,33 @@
 </xliff>
 {% endblock main %}
 
-{% if false %} {# avoid output of block directly #}
```

Comment:
This did also throw an error but was also unnecessary as it would never be output directly as it extends from another template.